spec/features/user_sees_homepage_spec.rb

```diff
@@ -19,19 +19,49 @@
     expect(page).to have_content("Register")
   end
 
-  it 'has a destinations link' do
+  it 'has a destinations link that works' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.